Stop debug console from going under sidebar. (Issue #5976)

prasoon2211 opened the pull request:
The diff is self explanatory. The problem was because `position: absolute` was being used on sidebar. Fixes issue #5976

Commit:
Stops debug console from going under sidebar

prasoon2211 commented:
Somethings seems wrong. Closing it for a bit :)

infograf768 commented:
This does not work as should indeed.
also you should patch the .less file and use generatecss.php

And RTL as well

TBH, I do not hnk this is a big issue: we can easily close the slidebar to see the debug

prasoon2211 commented:
Okay. I'll send in a fix soon.

prasoon2211 commented:
So I tried making the fix, which broke the view when sidebar is hidden. So, I fixed that as well. But then, the debug bar behaviour became different from that of `rtl` templates. I fixed that as well and now, the transitions are getting a bit of overlap sometimes.

Here's the [pasted patch](http://pastebin.com/CUZTCT0g). Anyone who's good at CSS can give it a try as I don't know how to fix the transition problems.
